helm/botkube/templates/global-config.yaml

```diff
@@ -13,6 +13,9 @@ data:
     executors:
       {{- .Values.executors | toYaml | nindent 6 }}
```

Comment:
`-` is this somehow not needed here? In your PR description, there is un expected dashes in Describe resource action which causes a wrong behavior. I removed dashes from curl brackets and it worked

Reply:
It renders correctly in this place:
```yaml
  global_config.yaml: |
    executors:
      kubectl-allow-all:
        kubectl:
          commands:
            resources:
            - deployments
            verbs:
            - get
          enabled: true
          namespaces:
            include:
            - .*
      kubectl-exec-cmd:
        kubectl:
          commands:
            verbs:
            - exec
          enabled: false
          namespaces:
            include:
            - botkube
            - default
          restrictAccess: false
      kubectl-not-bound-to-any-channel:
        kubectl:
          commands:
            resources:
            - deployments
            verbs:
            - port-forward
          enabled: true
          namespaces:
            include:
            - .*
      kubectl-read-only:
        kubectl:
          commands:
            resources:
            - deployments
            - pods
            - namespaces
            - daemonsets
            - statefulsets
            - storageclasses
            - nodes
            - configmaps
            - services
            - ingresses
            verbs:
            - api-resources
            - api-versions
            - cluster-info
            - describe
            - explain
            - get
            - logs
            - top
          defaultNamespace: default
          enabled: true
          namespaces:
            exclude: []
            include:
            - botkube
            - default
          restrictAccess: false
      kubectl-wait-cmd:
        kubectl:
          commands:
            verbs:
            - wait
          enabled: true
          namespaces:
            include:
            - botkube
            - default
          restrictAccess: false

    actions:
      get-created-resource:
        bindings:
          executors:
          - kubectl-read-only
          sources:
          - k8s-pod-create-events
          - k8s-all-events
        command: kubectl get {{ .Event.TypeMeta.Kind | lower }}{{ if .Event.Namespace }}
          -n {{ .Event.Namespace }}{{ end }} {{ .Event.Name }}
        displayName: Get created resource
        enabled: true
      show-created-resource:
        bindings:
          executors:
          - kubectl-read-only
          sources:
          - k8s-create-events
        command: kubectl describe {{ .Event.TypeMeta.Kind | lower }}{{ if .Event.Namespace
          }} -n {{ .Event.Namespace }}{{ end }} {{ .Event.Name }}
        displayName: Display created resource
        enabled: false
      show-logs-on-error:
        bindings:
          executors:
          - kubectl-read-only
          sources:
          - k8s-err-with-logs-events
        command: kubectl logs {{ .Event.TypeMeta.Kind | lower }}/{{ .Event.Name }} -n {{
          .Event.Namespace }}
        displayName: Show logs on error
        enabled: false
```

You're right in PR desc I had unnecessary dashes - I've actually fixed them already in the Helm chart `values.yaml`, but forgot to update in PR desc. Sorry for that!